pyvelox/CMakeLists.txt

```diff
@@ -11,24 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 if(VELOX_BUILD_PYTHON_PACKAGE)
   message("Creating pyvelox module")
-  include_directories(SYSTEM ${CMAKE_SOURCE_DIR})
+  include_directories(SYSTEM ${CMAKE_SOURCE_DIR} ${Protobuf_INCLUDE_DIRS}
+                      ${CMAKE_SOURCE_DIR}/velox/external/xxhash)
   add_definitions(-DCREATE_PYVELOX_MODULE -DVELOX_DISABLE_GOOGLETEST)
   # Define our Python module:
-  pybind11_add_module(pyvelox MODULE pyvelox.cpp serde.cpp signatures.cpp)
+  pybind11_add_module(pyvelox MODULE pyvelox.cpp serde.cpp signatures.cpp
+                      substrait.cpp)
   # Link with Velox:
   target_link_libraries(
     pyvelox
     PRIVATE velox_type
             velox_vector
+            velox_vector_test_lib
```

Comment:
You're correct this one is not needed. Removing it.

Reply:
Oh my mistake, this was used in the following way: 

1. Execute a Velox plan test utils
2. Create splits using test utils

I used this in the initial draft, but now I think it would be better to write separate utils for the Python module. Using test APIs shouldn't happen here.